src/WinRT.Runtime/ComWrappersSupport.net5.cs

```diff
@@ -215,12 +215,45 @@ public static void InitializeComWrappers(ComWrappers wrappers = null)
         
         private static Func<IInspectable, object> CreateFactoryForImplementationType(string runtimeClassName, Type implementationType)
         {
+            ParameterExpression[] parms = new[] { Expression.Parameter(typeof(IInspectable), "inspectable") };
+
+            var genericType = implementationType.GetGenericTypeDefinition();
```

Comment:
Should this be moved within the if IsGenericType as otherwise it doesn't need to be retrieved right?